Source/Charts/Charts/PieChartView.swift

```diff
@@ -314,7 +317,7 @@ open class PieChartView: PieRadarChartViewBase
     /// 
     /// - note: Use holeTransparent with holeColor = nil to make the hole transparent.*
     @objc open var holeColor: NSUIColor?
-    {
+        {
```

Comment:
code style issue for aligning {

Reply:
I didn't did anything special here just auto indented all the code

Reply:
ok, you can have different code style on your side, but you have to keep the code style consistent across the library. Here 
![image](https://user-images.githubusercontent.com/4375169/35542881-4408de7a-059d-11e8-8a7e-597da9908cc1.png)
the `{` has more indent than the old one. Need to fix all code style issues before merge.

Reply:
ok will fix it